### PR TITLE
Dev UI

### PR DESCRIPTION
**Pull request by IvanPadalka**
Addidn changes

### CODE REVIEW

**Comment by IvanPadalka** [comment]
Some changes

File: src/main/java/com/dimsirka/animalservice/controllers/AnimalController.java [hunk 1]
```diff
@@ -77,6 +78,11 @@ public String getAll(@RequestParam(name = "page", required = false, defaultValue
         return "animal/list";
     }
 
+    @GetMapping({"/about"})
+    public String getAll() {
+        return "animal/about";
+    }
+
     @GetMapping("/animals")
```
Comment (MarrMika) [inline src/main/java/com/dimsirka/animalservice/controllers/AnimalController.java:86]: Please, create a separate controller HomeController and place this endpoint here.

File: src/main/resources/templates/order/create.ftl [hunk 2]
```diff
@@ -1,4 +1,354 @@
-<#include "../header.ftl">
+<!DOCTYPE html>
+<#assign sec=JspTaglibs["http://www.springframework.org/security/tags"]/>
+<html lang="en-US"
+      class=" js flexbox canvas canvastext webgl no-touch geolocation postmessage websqldatabase indexeddb hashchange history draganddrop websockets rgba hsla multiplebgs backgroundsize borderimage borderradius boxshadow textshadow opacity cssanimations csscolumns cssgradients cssreflections csstransforms csstransforms3d csstransitions fontface generatedcontent video audio localstorage sessionstorage webworkers no-applicationcache svg inlinesvg smil svgclippaths js_active  vc_desktop  vc_transform  vc_transform "
+      style="height: auto; overflow: auto;">
```
Comment (MarrMika) [inline src/main/resources/templates/order/create.ftl:5]: There is no need to rewrite the create page. Revert these changes!

File: src/main/resources/templates/animal/list.ftl [hunk 3]
```diff
@@ -1,16 +1,438 @@
-<#include "../header.ftl">
-<#assign pageSubmissionUrl = animalStatus???then('&animalStatus=${animalStatus}','') >
-<div class="container">
+<!DOCTYPE html>
+<#assign sec=JspTaglibs["http://www.springframework.org/security/tags"]/>
+<html lang="en-US"
+      class=" js flexbox canvas canvastext webgl no-touch geolocation postmessage websqldatabase indexeddb hashchange history draganddrop websockets rgba hsla multiplebgs backgroundsize borderimage borderradius boxshadow textshadow opacity cssanimations csscolumns cssgradients cssreflections csstransforms csstransforms3d csstransitions fontface generatedcontent video audio localstorage sessionstorage webworkers no-applicationcache svg inlinesvg smil svgclippaths js_active  vc_desktop  vc_transform  vc_transform "
+      style="height: auto; overflow: auto;">
```
Comment (MarrMika) [inline src/main/resources/templates/animal/list.ftl:5]: There is no need to rewrite  the list page. Revert these changes!

**Review by MarrMika — CHANGES_REQUESTED** [review]
Looks good, but we need enhancement

**Comment by MarrMika** [comment]
1. Change background photo
2.Add href=/animals?animalStatus=FREE to 'Більше'
3.Move an about part to the home page
4.Enhance footer with additional information

File: src/main/resources/templates/animal/list.ftl (hunk 3 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/list.ftl]: Cnages reverted

File: src/main/resources/templates/order/create.ftl (hunk 2 above)
Comment (IvanPadalka) [reply src/main/resources/templates/order/create.ftl:5]: Changes reverted

File: src/main/java/com/dimsirka/animalservice/controllers/AnimalController.java (hunk 1 above)
Comment (IvanPadalka) [reply src/main/java/com/dimsirka/animalservice/controllers/AnimalController.java:86]: Created Separated Controller

**Review by IvanPadalka — COMMENTED** [review]
All done

File: src/main/resources/templates/animal/about.ftl [hunk 4]
```diff
@@ -0,0 +1,1226 @@
...
+        #rs-demo-id {
+        }
+
+    </style>
+    <link rel="stylesheet" href="https://bridge82.qodeinteractive.com/wp-content/cache/minify/2bf8a.css" media="all">
+    <!--[if lt IE 9]>
+    <link rel='stylesheet' id='vc_lte_ie9-css'
+          href='https://bridge82.qodeinteractive.com/wp-content/plugins/js_composer/assets/css/vc_lte_ie9.min.css'
+          type='text/css' media='screen'/>
+    <![endif]-->
```
Comment (MarrMika) [inline src/main/resources/templates/animal/about.ftl:231]: What a hard code?

File: src/main/resources/templates/animal/about.ftl [hunk 5]
```diff
@@ -0,0 +1,1226 @@
...
+                    .vc_custom_1492524801071 {
+                        padding-right: 0px !important;
+                        padding-left: 0px !important;
+                    }
+
+                    .vc_custom_1481540011171 {
+                        padding-top: 67px !important;
+                        padding-bottom: 106px !important;
+                    }
+                </style>
```
Comment (MarrMika) [inline src/main/resources/templates/animal/about.ftl:582]: Take all style parts out of the main html code

File: src/main/resources/templates/animal/about.ftl [hunk 6]
```diff
@@ -0,0 +1,1226 @@
...
+                <script>
+                    var page_scroll_amount_for_sticky = 900;
+
+                </script>
```
Comment (MarrMika) [inline src/main/resources/templates/animal/about.ftl:586]: Take script out as well

File: src/main/resources/templates/animal/about.ftl [hunk 7]
```diff
@@ -0,0 +1,1226 @@
...
+                            .aboutus-section {
+                                padding: 90px 0;
+                            }
+
+                            .aboutus-title {
+                                font-size: 30px;
+                                letter-spacing: 0;
+                                line-height: 32px;
+                                margin: 0 0 39px;
+                                padding: 0 0 11px;
```
Comment (MarrMika) [inline src/main/resources/templates/animal/about.ftl:665]: Take this style section out as well

File: src/main/resources/templates/animal/about.ftl [hunk 8]
```diff
@@ -0,0 +1,1226 @@
...
+                "closeText": "Close",
+                "currentText": "Today",
+                "monthNames": ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"],
+                "monthNamesShort": ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"],
+                "nextText": "Next",
+                "prevText": "Previous",
+                "dayNames": ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"],
+                "dayNamesShort": ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"],
+                "dayNamesMin": ["S", "M", "T", "W", "T", "F", "S"],
+                "dateFormat": "MM d, yy",
```
Comment (MarrMika) [inline src/main/resources/templates/animal/about.ftl:1191]: What is the purpose of this part of the code?

File: src/main/resources/templates/animal/about.ftl [hunk 9]
```diff
@@ -0,0 +1,1226 @@
...
+
+                                            <div class="vc_empty_space" style="height: 15px">
+                                                    <span
+                                                        class="vc_empty_space_inner">
+                                                        <span
+                                                            class="empty_space_image">
+                                                        </span>
+                                                        </span>
+                                                    </div>
+                                            <a href="https://www.google.com/maps/place/%D0%91%D0%9E+%D0%94%D1%96%D0%BC+%D0%A1%D1%96%D1%80%D0%BA%D0%B0+.+%D0%9F%D1%80%D0%B8%D1%82%D1%83%D0%BB%D0%BE%D0%BA/@48.8942963,24.7626555,17z/data=!3m1!4b1!4m5!3m4!1s0x4730c391910041fd:0x6a789a2223e12e2d!8m2!3d48.8942963!4d24.7648442">
```
Comment (MarrMika) [inline src/main/resources/templates/animal/about.ftl:1077]: Make footer more informal
Delete useless and  nonactual information
Add gmap visualisation

File: src/main/resources/templates/animal/about.ftl [hunk 10]
```diff
@@ -0,0 +1,1226 @@
...
+                                                    <a href="#">Реквізити оплати</a></li>
+                                                <li id="menu-item-47"
+                                                    class="menu-item menu-item-type-custom menu-item-object-custom menu-item-47">
+                                                    <a href="#">Пошук</a></li>
+                                                <li id="menu-item-47"
+                                                    class="menu-item menu-item-type-custom menu-item-object-custom menu-item-47">
+                                                    <a href="#">Партнери</a></li>
+                                                <li id="menu-item-47"
+                                                    class="menu-item menu-item-type-custom menu-item-object-custom menu-item-47">
+                                                    <a href="#">Новини</a></li>
```
Comment (MarrMika) [inline src/main/resources/templates/animal/about.ftl:1058]: Add only actual items

File: src/main/resources/templates/animal/about.ftl [hunk 11]
```diff
@@ -0,0 +1,1226 @@
...
+            var f = d.getElementsByTagName(s)[0],
+                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : '';
+            j.async = true;
+            j.src =
+                '//www.googletagmanager.com/gtm.' + 'js?id=' + i + dl;
+            f.parentNode.insertBefore(j, f);
+        })(window, document, 'script', 'dataLayer', 'GTM-WCC77WV');//]]>
+
+    </script>
+
```
Comment (MarrMika) [inline src/main/resources/templates/animal/about.ftl:271]: Do we need this js script? What is the purpose of this part?

File: src/main/resources/templates/animal/about.ftl [hunk 12]
```diff
@@ -0,0 +1,1226 @@
...
+            border: none !important;
+            box-shadow: none !important;
+            height: 1em !important;
+            width: 1em !important;
+            margin: 0 .07em !important;
+            vertical-align: -0.1em !important;
+            background: none !important;
+            padding: 0 !important;
+        }
+
```
Comment (MarrMika) [inline src/main/resources/templates/animal/about.ftl:211]: Take this style section out as well

File: src/main/resources/templates/animal/about.ftl [hunk 13]
```diff
@@ -0,0 +1,1226 @@
...
+            "source": {"concatemoji": "https:\/\/bridge82.qodeinteractive.com\/wp-includes\/js\/wp-emoji-release.min.js?ver=5.3"}
+        };
+        !function (e, a, t) {
+            var r, n, o, i, p = a.createElement("canvas"), s = p.getContext && p.getContext("2d");
+
+            function c(e, t) {
+                var a = String.fromCharCode;
+                s.clearRect(0, 0, p.width, p.height), s.fillText(a.apply(this, e), 0, 0);
+                var r = p.toDataURL();
+                return s.clearRect(0, 0, p.width, p.height), s.fillText(a.apply(this, t), 0, 0), r === p.toDataURL()
```
Comment (MarrMika) [inline src/main/resources/templates/animal/about.ftl:163]: Take this js section out as well

File: src/main/resources/templates/animal/about.ftl [hunk 14]
```diff
@@ -0,0 +1,1226 @@
...
+        }
+
+        .vc_custom_1472637221526 {
+            padding-top: 58px !important;
+        }
+
+        .vc_custom_1474273042405 {
+            padding-top: 52px !important;
+        }
+
```
Comment (MarrMika) [inline src/main/resources/templates/animal/about.ftl:294]: Take this style section out as well

**Comment by MarrMika** [comment]
1. Enlarge logo image
2. Fill the footer out with actual information
3. Add google map visualization in footer

File: src/main/resources/templates/animal/about.ftl (hunk 8 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/about.ftl:1191]: Deleted

File: src/main/resources/templates/animal/about.ftl (hunk 7 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/about.ftl:665]: Moved to footer.ftl

File: src/main/resources/templates/animal/about.ftl (hunk 6 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/about.ftl:586]: Moving makes this script unworking

File: src/main/resources/templates/animal/about.ftl (hunk 5 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/about.ftl:582]: Taked

Comment (IvanPadalka) [reply src/main/resources/templates/animal/about.ftl:582]: Taken

File: src/main/resources/templates/animal/about.ftl (hunk 8 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/about.ftl:1191]: Deleted

File: src/main/resources/templates/animal/about.ftl (hunk 10 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/about.ftl:1058]: Some items deleted

File: src/main/resources/templates/animal/about.ftl (hunk 12 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/about.ftl:211]: taken

File: src/main/resources/templates/animal/about.ftl (hunk 4 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/about.ftl:231]: Deleted

File: src/main/resources/templates/animal/about.ftl (hunk 13 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/about.ftl:163]: Deleted

File: src/main/resources/templates/animal/about.ftl (hunk 14 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/about.ftl:294]: Moved

File: src/main/resources/templates/animal/about.ftl (hunk 6 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/about.ftl:586]: Not  moved and deleted. Its speed of scrolling

File: src/main/resources/templates/animal/about.ftl (hunk 7 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/about.ftl:665]: Moved

File: src/main/resources/templates/animal/about.ftl (hunk 11 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/about.ftl:271]: Deleted

File: src/main/resources/templates/animal/about.ftl (hunk 9 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/about.ftl:1077]: Done
